test/guided_proposal_test.rb

```diff
@@ -27,13 +27,89 @@
 require_relative "support/proposal_context"
 
 describe Y2Storage::GuidedProposal do
-  describe "#propose" do
-    using Y2Storage::Refinements::SizeCasts
+  using Y2Storage::Refinements::SizeCasts
+  let(:architecture) { :x86 }
+
+  include_context "proposal"
+
+  describe ".initial" do
```

Comment:
Good tests! What about adding also another to verify that it uses `ProposalSettings.new_for_current_product` by default?